_layouts/default.html

```diff
@@ -56,16 +56,17 @@
     owl.owlCarousel({
       loop: true,
       margin: 10,
-      autoplay: true,
       autoplayTimeout: 6000,
       autoplayHoverPause: true,
       responsiveClass: true,
       responsive: {
         0: {
           items: 1,
+          autoplay: true,
```

Comment:
what does this change do?

Reply:
If a desktop, autoplay is not desirable because the screen would display all the cards, in our case, the three of them. But when we have fewer cards, which we can witness on a mobile or iPad, autoplay needs to be enabled.